client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -63,19 +62,29 @@ const JetpackFAQ: React.FC = () => {
 						}
 					) }
 				</FoldableFAQ>
-
+				<FoldableFAQ
+					id="volume-discount"
+					question={ translate( 'Do you have any discounts for multiple sites?' ) }
+					onToggle={ onFaqToggle }
+				>
+					{ translate(
+						'Anyone with at least five websites can join our licensing platform and enjoy a 25% discount across all Jetpack products! You can learn more about our {{agenciesLink}}licensing platform and agency program here{{/agenciesLink}}.',
```

Comment:
@rcanepa, would you know if that 25% percent is stored in a variable in Calypso already?

Reply:
Not to my knowledge. I think this is the first mention of this discount on Calypso, but I could be wrong.

Reply:
Thanks for your input Renzo.